app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -68,7 +72,9 @@ fun UnioApp() {
     navigation(startDestination = Screen.WELCOME, route = Route.AUTH) {
       composable(Screen.WELCOME) { WelcomeScreen(navigationActions) }
       composable(Screen.EMAIL_VERIFICATION) { EmailVerificationScreen(navigationActions) }
-      composable(Screen.ACCOUNT_DETAILS) { AccountDetails(navigationActions) }
+      composable(Screen.ACCOUNT_DETAILS) {
+        AccountDetails(navigationActions, userRepositoryFirestore)
```

Comment:
Later on, this might need to be changed to a `UserViewModel`. I'm currently working on it and I'll adapt this as soon as possible.